### PR TITLE
Updating slack links for issue #44

### PR DESCRIPTION
**Pull request by kclibrarian**
## What does this PR do / why we need it
The slack links were outdated in the header and footer and need to link to current, persistent link.

## Which issue(s) does this PR fix
44: 2 slack links are expired.

Fixes #?
Updated links to new link.

## PR acceptance criteria

- [ ] Unit Tests
- [ ] E2E Tests
- [ ] Documentation

### CODE REVIEW

File: apps/landing-page/pages/community.mdx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export function CommunityHeader() {
 
 ## Join the community
 
-Start by joining our community. We have a [janus-idp](https://join.slack.com/t/janus-idp/shared_invite/zt-1pxtehxom-fCFtF9rRe3vFqUiFFeAkmg) Slack workspace that you can join and meet other Janus users. In all our communications, on whatever platform, we expect everyone to follow a code of contact.
+Start by joining our community. We have a [janus-idp](https://janus-idp.slack.com/archives/C04EDTPJRK5) Slack workspace that you can join and meet other Janus users. In all our communications, on whatever platform, we expect everyone to follow a code of contact.
```
Comment (serenamarie125) [inline apps/landing-page/pages/community.mdx:23]: https://join.slack.com/t/janus-idp/shared_invite/zt-1pxtehxom-fCFtF9rRe3vFqUiFFeAkmg is the proper link

File: packages/ui/components/footer/footer.tsx [hunk 2]
```diff
@@ -67,7 +67,7 @@ const navigation = {
     },
     {
       name: 'Slack',
-      href: 'https://join.slack.com/t/janus-idp/shared_invite/zt-1nii16o6e-SGscZ4YtAktL6rRtZZBUfA',
+      href: 'https://janus-idp.slack.com/archives/C04EDTPJRK5',
```
Comment (serenamarie125) [inline packages/ui/components/footer/footer.tsx:70]: https://join.slack.com/t/janus-idp/shared_invite/zt-1pxtehxom-fCFtF9rRe3vFqUiFFeAkmg is the proper link

**Review by serenamarie125 — CHANGES_REQUESTED** [review]
@kclibrarian thanks for your PR!  Just verified that https://join.slack.com/t/janus-idp/shared_invite/zt-1pxtehxom-fCFtF9rRe3vFqUiFFeAkmg is the proper link which does not expire!  We had a previous PR which updated the link in the header, but not the footer.  Please update and we will merg!

File: apps/landing-page/pages/community.mdx (hunk 1 above)
Comment (kclibrarian) [reply apps/landing-page/pages/community.mdx]: Updated!

**Review by christophe-f — APPROVED** [review]
LGTM

**Pull request merged**